backend/functions/utils.py

```diff
@@ -0,0 +1,152 @@
+import requests
+from users.utils import LANG_NAME_TO_CODE_ULCA, LANG_TRANS_MODEL_CODES
+
+### Utility Functions
+def get_batch_translations_using_indictrans_nmt_api(
+    sentence_list, source_language, target_language
+):
+
+    """Function to get the translation for the input sentences using the IndicTrans NMT API.
+
+    Args:
+        input_sentence (str): Sentence to be translated.
+        source_language (str): Original language of the sentence.
+        target_language (str): Final language of the sentence.
+
+    Returns:
+        list: List of dictionaries containing the translated sentences. 
+    """
+
+    # Get the translation model ID
+    model_id = LANG_TRANS_MODEL_CODES.get(f"{source_language}-{target_language}", 144)
+
+    # Convert language names to the language code
+    source_language = LANG_NAME_TO_CODE_ULCA[source_language]
+    target_language = LANG_NAME_TO_CODE_ULCA[target_language]
+
+    # Create the input sentences list
+    input_sentences = [{"source": sentence} for sentence in sentence_list]
+
+    headers = {
+        # Already added when you pass json= but not when you pass data=
+        # 'Content-Type': 'application/json',
+    }
+
+    json_data = {
+        "input": input_sentences,

```

Comment:
Hey, Aravinth mentioned that we can only send a max of 75 sentences per requests. So we'll have to loop through the `input_sentences` and call the API.

Something like this:
```py
output_sentences = []
for i in range(0, len(input_sentences), batch_size):
  batch_of_input_sentences = input_sentences[i:i+batch_size]
  # call the API for `batch_of_input_sentences`
  # append the output translations to an array
```

Reply:
Right and I will set the batch size as 75 at the moment then.

Reply:
Added this change as well. Please review.